content/127labs/drone-k8s-helm/index.md

```diff
@@ -0,0 +1,54 @@
+---
+date: 2017-06-10T00:00:00+00:00
+title: Kubernetes Helm
+author: 127labs
+tags: [ kubernetes, helm, docker ]
+repo: 127labs/drone-k8s-helm
+logo: kubernetes_helm.svg
```

Comment:
can you set to `helm.svg` since we already have a helm logo in the repository? thanks!